web/src/api/typings/jobs.ts

```diff
@@ -45,6 +45,8 @@ export type Job = {
     // TODO: check pipeline property
 };
 
+export type TJobWithExtendedAnnotators = Omit<Job, 'annotators'> & { annotators: TUserShort[] };
```

Comment:
We have already changed the description of the Job type, taking into account the received data, so these changes are redundant.

Reply:
Done, rolled back TJobWithExtendedAnnotators related changes